Fix: Skipped TestShareFile tests

nlok5923 opened the pull request:
Fixing the skipped TestShareFile test.

Kishan-Dhakan commented on tests/cli_tests/zboxcli_share_file_test.go:
```diff
@@ -1142,7 +1141,7 @@ func TestShareFile(t *testing.T) {
 		require.NotEmpty(t, initialReadPool)
 
-		require.Equal(t, 0.4, initialReadPool.Balance)
+		require.Equal(t, 0.4*1e10, float64(initialReadPool.Balance))
```
The readPool you are checking before and after downloading is of the owner. But the download is being done by `receiverWallet` here: `output, err = downloadFileForWallet(t, receiverWallet, configPath, downloadParams, false)`.

nlok5923 replied:
Fixed this in latest commit.

nlok5923 commented:
@Kishan-Dhakan the test are passing on test run here https://github.com/0chain/system_test/actions/runs/3216728675/jobs/5258912965 on https://github.com/0chain/0chain/pull/1721 pr but failing due authorizer crashing 
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/56120084/194798062-6a473cb1-1ecc-4756-be80-8faa89dc53b2.png">

stewartie4 reviewed (changes requested):
Few questions, why are these being skipped? Can you update the t.skip reason? 
Also why remove the logging?

nlok5923 commented:
> Few questions, why are these being skipped? Can you update the t.skip reason? Also why remove the logging?

Hi @stewartie4 
> why are these being skipped? 

Actually the test are being skipped because on sharing file and downloading file by receiver the read pool balance is not affected although it incurred some download cost due to downloading.

> Can you update the t.skip reason?

As now the test i fixed i think we won't be keeping the `t.Skip` statement. so shall i still update the reason.

> Also why remove the logging?

I had added those statement just for debugging purpose.

Kishan-Dhakan commented:
Failing due to unrelated authorizer crashes, merging this.
